decidim-budgets/app/views/decidim/budgets/order_summary_mailer/order_summary.html.erb

```diff
@@ -0,0 +1,20 @@
+<% if @component.try(:scope) %>
```

Comment:
This is in anticipation of #5941.

Reply:
Once components can be scoped, all components will return a scope with `component.scope` (can be its own when `scopes_enabled`, the parent component or the participatory space).

So `component.scopes_enabled` will return if the component is scoped or not.